src/reanimated2/animation/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './commonTypes';
+export * from './util';
+export * from './timing';
+export * from './spring';
+export * from './decay';
+export * from './delay';
+export * from './repeat';
+export * from './sequence';
+export * from './styleAnimation';
```

Comment:
I'm not the fan of this kind of imports. We started to export more types to share them between modules and those shouldn't be used by end users. IMO, the better approach would be to explicitly list things we want to export so we have more control of it.